Makefile

```diff
@@ -323,10 +324,15 @@ telemetrygen:
 .PHONY: update-otel
 update-otel:$(MULTIMOD)
 	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m stable --commit-hash $(OTEL_STABLE_VERSION)
-	git add . && git commit -s -m "[chore] multimod update stable modules"
+	git add . && git commit -s -m "[chore] multimod update stable modules" ; \
 	$(MULTIMOD) sync -s=true -o ../opentelemetry-collector -m beta --commit-hash $(OTEL_VERSION)
```

Comment:
What's the value of adding `; \` to make this into a single separated line? (Same for next two line too)

Reply:
The value here is, that if the STABLE / BETA - version has no updates the command does not get canceled by an error of git, because there is nothing to commit (e.g. someone updated something manually).

Reply:
Good to know, thanks for explaining!